src/mgmt/srv.c

```diff
@@ -409,6 +409,73 @@ process_join_request(Drpc__Call *drpc_req, Drpc__Response *drpc_resp)
 	D_FREE(resp);
 }
 
+/*
+ * Adds user and group name as owner to a new copy of the daos_prop_t passed in.
+ * The newly allocated prop is freed after pool create call returns.
+ */
+static int
+add_ownership_props(daos_prop_t **prop_out, char *owner, char *owner_grp)
```

Comment:
Comment is no longer accurate since we're creating a new prop and populating only with owner-user/owner-group.

Might be worthwhile to change the function name, too, maybe something like create_ownership_prop.

Reply:
thanks for the original template 👍